Source/WebSocket.swift

```diff
@@ -860,8 +879,9 @@ open class WebSocket : NSObject, StreamDelegate {
     /**
      Used to write things to the stream
      */
-    private func dequeueWrite(_ data: Data, code: OpCode, writeCompletion: (() -> ())? = nil) {
+    private func dequeueWrite(_ data: Data, code: OpCode, priority: Operation.QueuePriority = .normal, writeCompletion: (() -> ())? = nil) {
```

Comment:
Order is not maintained across priority in an OperationQueue so if you sent a low priority and then a high priority it may go ahead of the low priority.